docs/customization_checklist.md

```diff
@@ -1 +1,40 @@
-Want to use this for your own event? Here's what you need to change:
+## Welcome to EUI Event Template customization guide 
+
+If you want to use this template for your own event, here is everything that you need to change along with a brief description of how to make the respective change. The guide has been divided by sections of the web app:
+
+### Nav Bar:
+
+- [ ] **Change logo and favicon:** Follow the path ```eui-event-template/src/data/consts.js```. In the ```const.js``` file, on _line 12_, replace the placeholder rainbow cluster link with a link of your event’s logo.
+
+- [ ] **Change logo text:** Follow the path ```eui-event-template/src/utilities/rainbowLetters.jsx```. In the ```rainbowLetters.jsx``` file, on _line 27_, replace "{rainbowedLetters} SUMMIT" with the name of your event. 
+
+- [ ] **Change date and time:** Follow the path ```eui-event-template/src/data/consts.js```. In the ```const.js``` file, _within the sessionOne and sessionTwo objects_, there will be dateAndTime attributes. Replace the date and time with your event’s sessions. Keep the format intact.
+
+- [ ] **Change Add to Calendar link:** Follow the path ```eui-event-template/src/data/consts.js```. In the ```const.js``` file, _within the sessionOne and sessionTwo objects_, there will be calendarLink attributes. Replace the placeholder with your Add to Calendar link. 
+
+### Event Details:
+
+- [ ] **Replace hero image:** Follow the path ```eui-event-template/src/components/panels/EventDetails.jsx```. In the ```EventDetails.jsx``` file, _within the renderHero function (lines 13 - 21)_, replace the existing URL in the src attribute with the URL of your hero image. Replace the alt attribute’s text with text appropriate for your hero image. 
+
+- [ ] **Replace hero title and subtitle:** Follow the path ```eui-event-template/src/data/speakers.js```. In the ```speakers.js``` file, _within the speakers array (line 8)_, each speaker is listed as an object. Overwrite the placeholder information with your the actual information of your event’s speakers. If you want to add more speakers, add an additional object to the array. If you want to decrease the placeholder speakers, just remove the objects from the speakers array. 
+
```

Comment:
Might want to add another `### Speaker` header here like you did for Event Details and Talks sections